myproject/blog/views.py

```diff
@@ -1,34 +1,43 @@
-from django.core.paginator import Paginator
-from django.shortcuts import render
-from django.shortcuts import render
 from django.views.generic import TemplateView,ListView,detail
 
-from .models import Blogs,Authors
+from .models import Blogs,Authors,Comment
```

Comment:
Change model name of `Blogs` to `Blog` and `Authors` to `Author`

Reply:
Okay